app/assets/javascripts/3dbio_viewer/src/webapp/components/protvista/Protvista.module.css

```diff
@@ -1,8 +1,30 @@
+body {
```

Comment:
References to global components like `body` should go to App.css. On modules, only classes within the component. I'll move it.